include/glow/Support/Support.h

```diff
@@ -65,6 +65,21 @@ Stream &operator<<(Stream &os, const llvm::ArrayRef<E> list) {
   return os;
 }
 
+/// \returns a truncated version of the input string \p str to a maximum
+/// length of \p length. If the string effectively needs truncation then a
+/// given \p suffix will pe appended to the truncated string to signal that
+/// string was actually truncated.
+inline std::string truncateString(const std::string &str, size_t length,
+                                  const std::string &suffix = "...") {
+  return (str.size() > length) ? (str.substr(0, length) + suffix) : str;
+}
```

Comment:
This one can be removed now since looks like it's unused?

Reply:
Done.